scalafmt-tests/src/test/resources/rewrite/AvoidInfix2.stat

```diff
@@ -71,6 +71,6 @@ behaviour of "..." {
 behavior.of("..." {
```

Comment:
How is `behavior` actually matched here? The exclude filter has `behaviour`

Reply:
ach makes sense! I was thinking the other way around

Reply:
i know... i am frequently confused myself with the double negative ("never" and "exclude"), plus a little bit with `neverInfix` when used with `AvoidInfix` :)